### DISCUSSION TITLE
Could you explain what is the logic of deselecting selected TreeViewItem?

### DISCUSSION THREAD

**Discussion by alderic** [microsoft/fluentui#28251] [Q&A]
Hello, 
When clicking on the selected [treeviewitem](https://explore.fast.design/components/fast-tree-view), the element cancels the selection, is it a bug or a feature?
although in[ this example](https://learn.microsoft.com/en-us/fluent-ui/web-components/components/tree-view?pivots=typescript) it works as usual, but here version 2.0

**Comment by ValentinaKozlova** [comment]
Hi @bsunderhus, could you please add your input to this discussion?

**Reply by bsunderhus** [reply]
Hey @ValentinaKozlova, this is a FAST webcomponent element, unfortunately I'm only aware of the react v9 component implementation  https://react.fluentui.dev/?path=/docs/preview-components-tree-tree--default

**Reply by bsunderhus** [reply]
Maybe someone from @microsoft/fui-wc might be more capable on answering this
